### ISSUE TITLE
[BUG] forecasting in sample and out of sample with prediction intervals in ARIMA/AutoARIMA

### ISSUE THREAD

**Issue by IlyasMoutawwakil** [alan-turing-institute/sktime#1441]
> **Describe the bug**

There's a small problem with ARIMA/AutoARIMA when forecasting in sample and out of sample with prediction intervals. In the _pmdarima adapter, when `return_pred_int` is False, `_predict_in_sample` returns a pd.Series but when it's True, it returns a tuple (pd.Series, pd.DataFrame) which raises the append error.

> **To Reproduce**

```python
from sktime.forecasting.arima import ARIMA
import pandas as pd
import numpy as np

y_train = pd.Series(np.arange(1,21), index=pd.date_range(start='1/1/2021', end='20/1/2021'))

arima_forecaster = ARIMA(order=(1, 1, 1), suppress_warnings=True)
arima_forecaster.fit(y_train)
arima_forecaster.predict(np.array([-18, -14, -10, 5]), return_pred_int=True)
```

> **Expected behavior**

To return predictions with prediction intervals

> **Additional context**

Traceback :
```
AttributeError                            Traceback (most recent call last)
<ipython-input-10-d49673c473fc> in <module>()
----> 1 arima_forecaster.predict(np.array([-18, -14, -10, 5]), return_pred_int=True)

1 frames
/usr/local/lib/python3.7/dist-packages/sktime/forecasting/base/_base.py in predict(self, fh, X, return_pred_int, alpha)
    240             X=X_inner,
    241             return_pred_int=return_pred_int,
--> 242             alpha=alpha,
    243         )
    244 

/usr/local/lib/python3.7/dist-packages/sktime/forecasting/base/adapters/_pmdarima.py in _predict(self, fh, X, return_pred_int, alpha)
     69             y_ins = self._predict_in_sample(fh_ins, **kwargs)
     70             y_oos = self._predict_fixed_cutoff(fh_oos, **kwargs)
---> 71             return y_ins.append(y_oos)
     72 
     73     def _predict_in_sample(

AttributeError: 'tuple' object has no attribute 'append'
```

> **Versions**

<details>

I've run this one on a clean colab notebook after a `!pip install sktime[all_extras]`

</details>

**Comment by mloning** [comment]
Thanks for raising the issue @IlyasMoutawwakil - we're currently working on refactoring the interface for probabilistic forecasting interval x(see #1421). Help would be much appreciated for fixing the individual forecasters once #1421 is merged.

**Comment by fkiraly** [comment]
fixed by #1442.
